Source/ACE.Entity/AceObjectPropertiesSpell.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using ACE.Common;
+using MySql.Data.MySqlClient;
+namespace ACE.Entity
+{
+    [DbTable("ace_object_properties_spell")]
```

Comment:
@ddevec an AceObject can have many spells on it, these are different than the primary Spell which comes from DID table. As is currently the case, PropertyInts can only have a single entry per.